src/plugins/vis_index_pattern_selector/public/components/change_indexpattern.test.tsx

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
We can use the simplified header for new files:

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/.eslintrc.js#L36-L41

Reply:
Good catch.